common/culture/traditions/00_regional_traditions.txt

```diff
@@ -476,26 +502,39 @@ tradition_horse_lords = {
 
 	is_shown = {
 		OR = {
-			like_mongolic_group_trigger = yes
-			like_turkic_group_trigger = yes
+			# Warcraft
+			has_cultural_pillar = heritage_yaungolic
+			has_cultural_pillar = heritage_zaetaric
+			has_cultural_pillar = heritage_tharian
+			has_cultural_pillar = heritage_cenaric
+			has_cultural_pillar = heritage_orcish
+			has_cultural_pillar = heritage_agamagganic
 		}
 	}
 	
 	can_pick = {
-		custom_description = {
-			text = culture_in_steppe_desc
-			always = no
+		# Warcraft
+		custom_tooltip = {
+			text = culture_in_plains_desc
 			any_culture_county = {
 				percent >= 0.4
-				any_county_province = {
-					steppe_region_trigger = yes
+				OR = {
+					any_county_province = { steppe_region_trigger = yes	}
+					any_county_province = {
+						OR = {
+							geographical_region = world_innovation_camels
+							geographical_region = custom_oasis
+							geographical_region = custom_drylands
+						}	
+					}
 				}
```

Comment:
Just steppe region may look better. Kezan is drylands.

Reply:
Replaced with `custom_deserts` and `world_kalimdor_central`. Removed `world_innovation_camels` as it is covered by `custom_deserts`.

Reply:
Feralas is center too.

Reply:
We have `world_steppe` AKA nomad regions basically. I would add `world_kalimdor_central_desolace` and `world_kalimdor_central_durotar` there and use it in this condition.

Reply:
Sounds good. Updated `world_steppe_west` accordingly. How about Mulgore? It's not dry or dusty but technically fits the definition of steppe. It is also already in `graphical_steppe`.